Tests/Fido2.Tests/TestFiles/attestationOptionsU2F.json

```diff
@@ -4,10 +4,10 @@
   "rp":{
     "name":"localhost",
     "id":"localhost"},
-  "user":{
-    "name":"aseigler",
-    "id":"ABC",
```

Comment:
NOTE: this is not a valid id, which should a valid Base64Url encoded byte array.